plugins/modules/net_tools/nmcli.py

```diff
@@ -106,10 +107,18 @@
             - A list of DNS search domains.
         elements: str
         type: list
+    method4:
+        description:
+            - Configuration method to be used for IPv4.
+            - If ip4 is set, ipv4.method is automatically set to manual and this parameter is not needed.
```

Comment:
```suggestion
            - If I(ip4) is set, C(ipv4.method) is automatically set to C(manual) and this parameter is not needed.
```
(If both changes are fine, please apply similar ones to the IPv6 versions.)